content/blog/online-privacy.en.md

```diff
@@ -0,0 +1,45 @@
+{
+    "title": "Essential browser extensions for more privacy",
+    "type": "blog",
+    "date": "2019-10-21T21:11:25+00:00",
+    "description": "Installing security-focused browser extensions that block pop-up advertisements and completely encrypt website traffic from being monitored or tracked will, definitely increase your privacy—and may even speed up your browser.",
+    "featured_image": "/img/blog/online-privacy.jpg",
+    "tags": [ "privacy", "online", "adds", "cookies", "https" ],
+    "authors": [ "nishantwrp" ]
+}
+
+Websites these days collect a lot of information about you. When you leave the website, they then want to know where you are going next. They leave tracking cookies on your browser. 
+
+<img src="/img/blog/online-privacy.jpg">
+
+Tools like Google Analytics, they register everything from your geographical location to how often to visit a webpage.
+
+Below are some privacy add-ons that give your browser extra muscles to keep you safe online. It starts with ad blocker extensions that allow users to block irritating and malicious ads, save bandwidth and opt out of intrusive marketing systems.
+
+## Privacy Badger
+
+[Privacy Badger](https://www.eff.org/privacybadger) is arguably the best anti-tracking web browser extension. Launched by Electronic Frontier Foundation (EFF), it’s a lightweight add-on with a user-friendly toolbar button that keeps a record of your data. The “green/yellow/red” block-list system shows which webpages are tracking you and gives you the ability to block any or all of it at any point. The one feature that sets Privacy Badger apart is that it doesn’t block all content, just the tracking scripts that follow you from one site to another. If the content tracking you is essential to a page’s functionality, Privacy Badger will allow it to load with the tracking capabilities blocked.
+
+Privacy Badger is available for Chrome, Opera, Firefox, and even Firefox on Android phones.
+
+## uBlock Origin
+
+[uBlock Origin](https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm?hl=en) is one of the most popular ad blocker extensions. It prevents pesky popups and widgets from sending your data to advertisers. uBlock Origin lets you create personalized lists which gives you more control over what ads to block and what ads to allow. It has also been found to effectively block more ads than others while using less memory, so it doesn’t slow down your browsing experience.
+
+Unlike ad blockers, anti-tracking extensions block cookies and protect your online anonymity by preventing tracking and malvertising of third-party advertisers.
+
+uBlock Origin is available for Chrome, Opera, Firefox, and Safari.
+
+## Ghostery
```

Comment:
There have been reports of Ghostery actually tracking users itself. Even if that is not the case anymore, I don't feel comfortable recommending it to users.